app/src/main/java/com/kodeco/android/coordplot/components/MapSlider.kt

```diff
@@ -0,0 +1,46 @@
+package com.kodeco.android.coordplot.components
+
+import android.text.Layout
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+//noinspection UsingMaterialAndMaterial3Libraries
+import androidx.compose.material.Slider
+//noinspection UsingMaterialAndMaterial3Libraries
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.R
+import com.kodeco.android.coordplot.theme.CoordPlotTheme
+
+@Composable
+fun MapSlider(titleRes: Int, percentage: Float, onPercentageChange: (Float) -> Unit) {
+    val textString = stringResource(id = titleRes)
+    Row(
+        modifier = Modifier
+            .padding(16.dp, 0.dp, 16.dp, 0.dp)
+    ) {
+        Text(
+            text ="$textString : ${String.format("%.2f",percentage * 100)} %",
+            modifier = Modifier.width(120.dp)
```

Comment:
Minor note - but same thing here, you could create a variable for this 120 value. Also the same for the padding values above and below.